src/amo/components/AddonReviewManager/index.js

```diff
@@ -82,23 +84,23 @@ export class AddonReviewManagerBase extends React.Component<InternalProps> {
     } = this.props;
 
     const isReply = review.isDeveloperReply;
-    const reviewGuideLink = i18n.sprintf(
-      i18n.gettext(
-        'Please follow our %(linkStart)sreview guidelines%(linkEnd)s.',
-      ),
-      {
-        linkStart: '<a href="/review_guide">',
-        linkEnd: '</a>',
-      },
-    );
 
-    /* eslint-disable react/no-danger */
+    const linkParts = getLocalizedTextWithLinkParts({
+      i18n,
+      text: 'Please follow our %(linkStart)sreview guidelines%(linkEnd)s.',
```

Comment:
You need to apply `i18n.gettext` to this string before passing it into `getLocalizedTextWithLinkParts` (or even while passing it in), otherwise it will never appear to translators as a string for translation.

Reply:
ha ..yes, thanks : )

Reply:
this explains why one of my tests wasn't working how I wanted